partiql-planner/src/main/kotlin/org/partiql/planner/transforms/RelConverter.kt

```diff
@@ -106,13 +108,22 @@ internal object RelConverter {
             rexOpStructField(k, v)
         }
         val op = rexOpStruct(fields)
-        rex(StaticType.STRUCT, op)
+        val type = StructType(
+            fields = fields.mapIndexed { i, it -> StructType.Field(schema[i].name, schema[i].type) },
+            contentClosed = true,
+            constraints = setOf(
+                TupleConstraint.Open(false)
+            )
+        )
```

Comment:
I see why this was added, but my logic for not doing this was that PlanTyper will rewrite it anyways with the appropriate field types. There's no harm in having it here too

Reply:
I've since removed the default constructor from the RelConverter and have moved it to the `NormalizeSelect`.